src/app/frontend/replicasetdetail/replicasetdetail.html

```diff
@@ -138,8 +138,8 @@
           </md-input-container>
           <md-input-container class="kd-replicasetdetail-option-picker">
             <label>Source</label>
-            <!-- TODO(maciaszczykm): Handle value change. -->
-            <md-select ng-model="ctrl.eventSource" required>
+            <md-select ng-model="ctrl.eventSource"
+                       ng-change="ctrl.handleEventFiltering()" required>
```

Comment:
@bryk It's done this way, because if we change filtering on type to 'Warning' and on source to 'User' and then switch source to 'System' for example, then initial events set needs to be filtered. I think using one filtering method is not that bad in this case, because we need to have intersection of two output sets. I will think about better solution.